cms/djangoapps/contentstore/views/item.py

```diff
@@ -784,7 +784,15 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
     #instead of adding a new feature directly into xblock-info, we should add them into override_type.
     override_type = {}
     if getattr(xblock, "is_entrance_exam", None):
-        override_type['is_entrance_exam'] = xblock.is_entrance_exam
+        override_type['is_entrance_exam_section'] = xblock.is_entrance_exam
```

Comment:
@chrisndodge 'is_entrance_exam_section' just indicating that is xblock either an entrance-exam-chapter or not. Used only for hiding entrance exam's delete and drag functionality.